dashboard/config/initializers/session_store.rb

```diff
@@ -1,10 +1,10 @@
 require 'cdo/cookie_helpers'
-require_relative '../custom_session_store/mirror_cookies_in_redis_store'
+require_relative '../custom_session_store/migrate_cookies_to_redis_store'
```

Comment:
Is mirror_cookies_in_redis_store still in use? Does that file need to go away here?

Reply:
It isn't; I had it in my head that I was going to clean that up later when I eventually remove the `MigrateCookiesStore` logic, but I guess there's no reason not to do so now